packages/animations/example/lib/fade_scale_transition.dart

```diff
@@ -16,16 +16,16 @@ class _FadeScaleTransitionDemoState extends State<FadeScaleTransitionDemo>
     with SingleTickerProviderStateMixin {
   AnimationController _controller;
 
-  bool _showFab = true;
-
   @override
   void initState() {
     _controller = AnimationController(
-      value: 1.0,
+      value: 0.0,
       duration: const Duration(milliseconds: 150),
       reverseDuration: const Duration(milliseconds: 75),
       vsync: this,
-    );
+    )..addListener(() {
+        setState(() {});
```

Comment:
nit: add a comment inside the setState closure explaining why we have this empty set state.